TodoApi/Dockerfile

```diff
@@ -0,0 +1,22 @@
+#See https://aka.ms/containerfastmode to understand how Visual Studio uses this Dockerfile to build your images for faster debugging.
+
+FROM mcr.microsoft.com/dotnet/aspnet:7.0 AS base
+WORKDIR /app
+EXPOSE 80
+EXPOSE 443
+
+FROM mcr.microsoft.com/dotnet/sdk:7.0 AS build
+WORKDIR /src
+COPY ["TodoApi/TodoApi.csproj", "TodoApi/"]
+RUN dotnet restore "TodoApi/TodoApi.csproj"
+COPY . .
+WORKDIR "/src/TodoApi"
+RUN dotnet build "TodoApi.csproj" -c Release -o /app/build
+
+FROM build AS publish
+RUN dotnet publish "TodoApi.csproj" -c Release -o /app/publish /p:UseAppHost=false
```

Comment:
Why `p:UseAppHost=false`?

Reply:
I set `p:UseAppHost` to false to disable the generation of an executable file (i.e. `TodoApi.exe`) not needed on Linux.

Reply:
Docker files were removed in favour of using dotnet publish container.